code/__DEFINES/sound.dm

```diff
@@ -73,3 +74,7 @@
 	'sound/hallucinations/growl3.ogg', 'sound/hallucinations/im_here1.ogg', 'sound/hallucinations/im_here2.ogg', 'sound/hallucinations/i_see_you1.ogg', 'sound/hallucinations/i_see_you2.ogg',\
 	'sound/hallucinations/look_up1.ogg', 'sound/hallucinations/look_up2.ogg', 'sound/hallucinations/over_here1.ogg', 'sound/hallucinations/over_here2.ogg', 'sound/hallucinations/over_here3.ogg',\
 	'sound/hallucinations/turn_around1.ogg', 'sound/hallucinations/turn_around2.ogg', 'sound/hallucinations/veryfar_noise.ogg', 'sound/hallucinations/wail.ogg')
+
+#define LAVALAND_OUTSIDE list('sound/ambience/mgs2/windy1.ogg', sound/ambience/mgs2/windy3.ogg')

```

Comment:
```suggestion
#define LAVALAND_OUTSIDE list('sound/ambience/mgs2/windy1.ogg', 'sound/ambience/mgs2/windy3.ogg')
```

Reply:
Having it modular would also be nice, if possible

Reply:
The sound files themselves could be modular by being placed in the face/sounds folder, but the defines can't since it'd mean making up an entirely new list and making the areas use that list instead. I mean, it'd work, but it'd be more complexity than it's worth and unless you do that for *all* area defines, it just ends up being plain weird.

Reply:
Could just copy defines to face and use that rather then the current one?

Reply:
> 
> 
> Having it modular would also be nice, if possible

[This should at least get you in the right direction, brush](https://github.com/brushtool/fpstation/blob/master/hippiestation/README.md#defines)